syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -974,7 +968,9 @@ def hyperband_rung_levels(
         ), f"max_t ({max_t}) must be greater than grace_period ({grace_period})"
         rf = reduction_factor
         min_t = grace_period
-        max_rungs = int(np.log(max_t / min_t) / np.log(rf) + 1)
+        max_rungs = 0
```

Comment:
This created a nasty bug, which works on Mac, but fails on CI system, I think due to round-off.

Reply:
Nice catch